deepdrivemd/misc/aggregator_reader.py

```diff
@@ -0,0 +1,193 @@
+import adios2
+import numpy as np
+from deepdrivemd.utils import t1Dto2D
+
+
+class ADIOS_READER:
```

Comment:
Add docstrings and type hinting to document function inputs and class methods.

Reply:
Remove noqa

Reply:
Done